glTF-BinExporter/GlTFExporterCommand.cs

```diff
@@ -89,8 +86,32 @@ protected override Result RunCommand(Rhino.RhinoDoc doc, RunMode mode)
                 return Result.Success;
             } catch (Exception e) {
                 RhinoApp.WriteLine("ERROR: Failed exporting selected geometry to file.");
+                System.Diagnostics.Debug.WriteLine(e.Message);
                 return Result.Failure;
             }
         }
+
+        private SaveFileDialog GetSaveFileDialog(bool binaryExport)

```

Comment:
🎉  Sweet!

Reply:
I've been thinking this probably needs cleaned up a bit more. We can infer the binary or text from the extension. We can add both options to the SaveFileDialog filter and go from there.